### PR TITLE
Added #12069: Add additional custom text functionality for print all assigned.

### LINKED ISSUE #12069: [Feature Request]:  Print all assigned - additional text
### Is your feature request related to a problem? Please describe.

There is no option for additional text under the Print All Assigned option.

### Describe the solution you'd like

Adding filed under /admin/branding, which can be used as an additional text in Print All Assigned. I am sharing with you images as a helper.

add custom text
final result

### Describe alternatives you've considered

Adding a simple field under the admin/branding where we can modify that custom text.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by nikomatjasic**
Fix #12069

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Under admin/branding add text under Additional Print Text. Go to user, Print All Assigned - above the sign and date the text should be visible.

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

* PHP version:  8.0.18 
* MySQL version: mariadb:10.6.4-focal
* Webserver version: Docker (Version 3.6.0)
* OS version: MacOS Mojave (10.14.6)

# Checklist:

- [x] I have read the Contributing documentation available here: https://snipe-it.readme.io/docs/contributing-overview
- [x] I have formatted this PR according to the project guidelines: https://snipe-it.readme.io/docs/contributing-overview#pull-request-guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by nikomatjasic** [comment]
Conflict with the translations. This part of the code is needed.
```
@if($settings->print_additional_text)
    <table>
        <tr>
            <td>
                {!! $settings->print_additional_text `!!}
            </td>
        </tr>
    </table>
@endif
```

File: resources/views/settings/branding.blade.php [hunk 1]
```diff
@@ -246,12 +226,24 @@
...
                             </div>
                         </div>
+                        <!-- Additional print -->
+                        <div class="form-group {{ $errors->has('print_additional_text') ? 'error' : '' }}">
+                            <div class="col-md-3">
+                                {{ Form::label('print_additional_text', trans('admin/settings/general.print_additional_text')) }}
+                            </div>
+                            <div class="col-md-9">
+                                @if (config('app.lock_passwords')===true)
+                                    {{ Form::textarea('footer_text', Request::old('print_additional_text', $setting->print_additional_text), array('class' => 'form-control', 'rows' => '4', 'placeholder' => 'Optional additional print text','disabled'=>'disabled')) }}
```
Comment (snipe) [inline resources/views/settings/branding.blade.php:236]: Copypasta?  Should reflect the field name.

File: resources/views/settings/branding.blade.php [hunk 2]
```diff
@@ -246,12 +226,24 @@
...
+                                    <p class="text-warning"><i class="fas fa-lock"></i> {{ trans('general.feature_disabled') }}</p>
+                                @else
+                                    {{ Form::textarea('print_additional_text', Request::old('print_additional_text', $setting->print_additional_text), array('class' => 'form-control','rows' => '4','placeholder' => 'Optional additional print text')) }}
+                                @endif
+                                <p class="help-block">{{ trans('admin/settings/general.print_additional_text_help') }}</p>
+                                {!! $errors->first('footer_text', '<span class="alert-msg" aria-hidden="true">:message</span>') !!}
```
Comment (snipe) [inline resources/views/settings/branding.blade.php:242]: Same as before - should reflect fieldname.

**Review by snipe — CHANGES_REQUESTED** [review]
Hey there - sorry for the delay in addressing this. This seems like a good start! 

I do have a few issues here, not so much with the tech part. I'd prefer a field name that is a little clearer. `print_additional_text` could mean anything, really. In this case, it pertains only to the print assigned part of things, but as someone who has to put eyes on the actual raw DB, I would have no idea what the field holds. Maybe something like `all_assigned_print_additional_text`? That still feels a bit clunky, but I'd take it for now.

There are also zero circumstances under which I'd take a PR where we `{!! $foo !!}` user entered input. We'd use the markdown helper here, a la the optional dashboard message: `{!!  Helper::parseEscapedMarkedown($snipeSettings->dashboard_message)  !!}`. Using the unescaped version would leave the system open to XSS attacks, and while it would require an escalated permission (superadmin) to do The Bad Things, we still just don't allow that within the system. 

A bit unclear on the multiple line breaks after the echo in the blade as well. 

I know it's been a bit since you opened this PR, but if you're still interested in seeing this merged, if you could fix up those small bits I mentioned above, I think this could be useful to some folks who use Snipe-IT. I know it sounds a little nit-picky, but we want to make sure this will scale and be flexible enough to grow with the project in the right ways.

**Comment by snipe** [comment]
Closing this for housekeeping. If you'd like to re-open it, we'd probably take it, with the changes above and also if it's targeted to `develop` instead of master, per our contributing docs.

**Pull request closed without merging**